### ISSUE TITLE
Option to browse album via song

### ISSUE THREAD

**Issue by onlytanmoy** [Sangwan5688/BlackHole#83]
When I click on the 3 dots in the song being played, it will show me option to go to the album the song is part of so Naaraaz in case of the below example screenshot 
![Screenshot_20210718-180333](https://user-images.githubusercontent.com/48493659/126075945-b1a98101-fe03-4481-a9d7-f2419ae1b13f.jpg)

**Comment by Sangwan5688** [comment]
It might not be possible for all songs. Some songs have their name as the album itself. And the API doesn't give that much info about a particular song at some places like from you play from a playlist on the home screen or from the artists' section. Anyway, I will try adding it.

**Comment by onlytanmoy** [comment]
Yeah, got your point. Recently I figured out that if I swipe up on 'Now Playing' card it will list me all the songs from that album, 'Naaraaz' in this case if I have played the song from that album.
Is that 'Now Playing' known as Image Slider? I hope not :)

**Comment by Sangwan5688** [comment]
'Now Playing' is a draggable sheet and it contains the list of songs which you are playing. Basically, it's the 'Now Playing Queue'. If you play a song from an album then it will show the rest songs there. If you play from the playlist then it will show playlists there. It's not `Image Slider`. Image Slider is the slider that contains the song image. The image you see on the playing screen is slidable. Try sliding it left or right. It will change the image as well as the song. Basically, it's an image slider that needs to keep in sync with the song.

**Comment by onlytanmoy** [comment]
Damn...I wasn't aware of this nifty Image Slider option until you told me now, glad I asked lol.
Although I understand you will remove Image Slider as mentioned in issue no. 84 and will look to bring it back with all issues ironed out.
Many thanks for the comprehensive explanation of Now Playing & Image Slider 💯
